conf/test.config

```diff
@@ -10,16 +10,51 @@
 params {
   config_profile_name = 'Test profile'
   config_profile_description = 'Minimal test dataset to check pipeline function'
+  
   // Limit resources so that this can run on Travis
   max_cpus = 2
   max_memory = 6.GB
   max_time = 48.h
+
+
+
   // Input data
-  // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-  // TODO nf-core: Give any required params for the test so that command line flags are not needed
-  singleEnd = false
   readPaths = [
-    ['Testdata', ['https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R1.tiny.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R2.tiny.fastq.gz']],
-    ['SRR389222', ['https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub1.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub2.fastq.gz']]
+    ['SRR8823360', ['https://github.com/grst/test-datasets/raw/smartseq2/testdata/SRR8823360_1.fastq.gz', 'https://github.com/grst/test-datasets/raw/smartseq2/testdata/SRR8823360_2.fastq.gz']],
+    ['Bcell', ['https://github.com/grst/test-datasets/raw/smartseq2/testdata/b-cell1_1.fastq.gz', 'https://github.com/grst/test-datasets/raw/smartseq2/testdata/b-cell1_2.fastq.gz']],
+    ['Tcell', ['https://github.com/grst/test-datasets/raw/smartseq2/testdata/t-cell1_1.fastq', 'https://github.com/grst/test-datasets/raw/smartseq2/testdata/t-cell1_2.fastq']],
   ]
+  fasta = 'https://github.com/grst/test-datasets/raw/smartseq2/testdata/ref/hg38_reduced.fa.gz'
+  gtf = 'https://github.com/grst/test-datasets/raw/smartseq2/testdata/ref/gencode_v30_reduced.gtf.gz'
+}
+
+//Override defaults to always terminate and match the CI resource limits. 
+process {
+  errorStrategy = 'ignore'
+  withName:makeSTARindex {
+    cpus = 2 
+    memory = "6.GB"
+  }
+  withName:STAR {
```

Comment:
We typically tend to use `withLabel` these days to keep the number of very similar default parameters / resource requests lower... See the rnaseq pipeline for example . You also don't need to do this here in the test profile, instead do it in the base.config please and keep this out of the test config :-)